packages/js-dpp/test/integration/identity/validation/validateIdentityFactory.spec.js

```diff
@@ -369,9 +369,9 @@ describe('validateIdentityFactory', () => {
   });
 
   it('should return valid result if an identity model is valid', () => {
-    const result = validateIdentity(rawIdentity);
+    const result = validateIdentity(identity);
```

Comment:
`validateIdentity` expects rawIdentity but not an instance of Identity.

Reply:
It's literally what the test name implies: should return valid result if an **identity model** is valid. There's another test for the raw identity right above that one

Reply:
Discussed it verbally: this function doesn't accept model anymore only raw and it seems this test is outdated and just needs to be removed.